components/script/script_thread.rs

```diff
@@ -1173,25 +1180,17 @@ impl ScriptThread {
     ///
     /// Attempt to update the rendering and then do a microtask checkpoint if rendering was actually
     /// updated.
-    pub(crate) fn update_the_rendering(&self, requested_by_renderer: bool, can_gc: CanGc) {
+    pub(crate) fn update_the_rendering(&self, can_gc: CanGc) {
         *self.last_render_opportunity_time.borrow_mut() = Some(Instant::now());
 
         // If the ScriptThread animation timer fired, this is an animation tick.
-        let mut is_animation_tick = requested_by_renderer;
-        if self
+        let is_animation_tick = self
             .should_trigger_script_thread_animation_tick
-            .load(Ordering::Relaxed)
-        {
+            .load(Ordering::Relaxed);
+        if is_animation_tick {
             self.should_trigger_script_thread_animation_tick
                 .store(false, Ordering::Relaxed);
-            *self.scheduled_script_thread_animation_timer.borrow_mut() = None;
-            is_animation_tick = true;
-        }
-
-        // If this is an animation tick, cancel any upcoming ScriptThread-based animation timer.
-        // This tick serves the purpose and we to limit animation ticks if some are coming from
-        // the renderer.
```

Comment:
Why remove this comment? It explains why we are canceling the timer. The new comment just repeats what the code does.